psyneulink/core/llvm/codegen.py

```diff
@@ -312,15 +322,24 @@ def gen_composition_exec(ctx, composition, *, tags:frozenset):
 
         zero = ctx.int32_ty(0)
         any_cond = ir.IntType(1)(0)
-
+        is_finished_flags = [None] * len(composition.nodes)
+        for idx, node in enumerate(composition.nodes):
+            #FIXME: This skips nested nodes
+            from psyneulink import Composition
```

Comment:
Do not import `Composition` here. Check for the presence of `is_finished_flag` instead.